src/XrdTpc/PMarkManager.cc

```diff
@@ -0,0 +1,72 @@
+//------------------------------------------------------------------------------
+// This file is part of XrdTpcTPC
+//
+// Copyright (c) 2013 by European Organization for Nuclear Research (CERN)
```

Comment:
Another quite minor thing, but this is off by 10 years :-)

Reply:
Actually, what's the policy for the headers of those files? It's quite inconsistent throughout the entire project :D

Reply:
Yes, it was likely just copied without change, sometimes with the right year, sometimes with the same as before...

Reply:
Alright then, done for the PMarkManager.cc :)